### DISCUSSION TITLE
Как извлечь символ переноса строки из поля текст?

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#375] [Как это сделать?] [answered]
Несрочное, может кто-то сталкивался?

При отправке нотификации через текст из текстового поля не ставятся переносы строк. Работает только через html с ручным добавлением <br> в тексте. _Потом, эти "br" приходится убирать, т.к. телеграм хорошо воспринимает скрытый символ переноса "\n", но выкидывает ошибку и не отправляет при наличии br._ 

<img width="1710" alt="Снимок экрана 2023-05-18 в 12 01 36" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/91674114/c879fd67-ef49-4108-acf5-12fca32931cd">

Вопрос: **как можно вытащить \n из текстового поля и заменить?** Либо может есть какой-то альтернативный вариант?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
1. Мы попробуем поправить, что бы `notificationSend` воспринимал многострочный текст.

2. Заменяем переносы строк на `br`:

```
= : strReplace(str: $str; from: $#nl; to: "<br>")
```
